e2e-test/tsps_gcp_e2e_test.py

```diff
@@ -19,7 +19,7 @@ def update_imputation_pipeline_workspace(tsps_url, workspace_project, workspace_
         "wdlMethodVersion": wdl_method_version
     }
 
-    uri = f"{tsps_url}/api/admin/v1/pipeline/imputation_beagle"
+    uri = f"{tsps_url}/api/admin/v1/pipeline/array_imputation"
```

Comment:
Should we update https://github.com/broadinstitute/dsp-reusable-workflows/blob/59d4c95a5cd432581d28cee8b2db0586ec6da403/e2e-test/resources/tsps/tsps_gcp_set_up_control_workspace.py#L129 as well?

Reply:
absolutely, great catch